### PR TITLE
[WIP] Initialize status component [After UI merge]

### LINKED ISSUE #1688: Create component for connecting, loading, loaded states
We currently use tm-data-connecting, tm-data-loading, tm-data-msg in almost all views to handle connection states. We should refactor those 3 into one component.

### PR DESCRIPTION
**Pull request by sabau**
Closes #1688 

_Description:_

- fix some css property in variables and app.css
- create managed-body component
- use it wallet and page transactions

❤️ Thank you!

---

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer

### CODE REVIEW

File: app/src/renderer/components/common/ManagedBody.vue [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+</template>
+
+<script>
+import { TmDataMsg, TmDataLoading, TmDataEmpty } from "@tendermint/ui"
+import DataEmptySearch from "common/TmDataEmptySearch"
+import TmDataError from "common/TmDataError"
+import ModalSearch from "common/TmModalSearch"
+import TmDataConnecting from "common/TmDataConnecting"
+export default {
+  name: `managed-body`,
```
Comment (jbibla) [inline app/src/renderer/components/common/ManagedBody.vue]: not sure i understand the naming choice here @sabau

Comment (sabau) [reply app/src/renderer/components/common/ManagedBody.vue]: I wasn't sure neither honestly.
I was thinking of the goal of this component, basically it try to manage the statuses of data you pass to it, with a mandatory body slot, so I tried to express it somehow but the naming is not really clear.

Do you have some suggestion on how to make it change it?

It manages all the statuses in one place so I thought of manage something

Comment (sabau) [reply app/src/renderer/components/common/ManagedBody.vue]: Oh wait, I just realized that this component will fit as sub component of tm-page, once the merge is complete, will simply define a new slot

**Comment by sabau** [comment]
I would like to re-open this in a cleaner manner (locally this grew too much)

**Pull request closed without merging**